frontend/src/pages/SettingsRouter/SettingsRouter.tsx

```diff
@@ -180,6 +213,27 @@ export const SettingsRouter = () => {
 							</NavLink>
 						))}
 					</Stack>
+					<Stack gap="0">
+						<Box mt="12" mb="10" ml="8">
+							<Text size="xxSmall" color="secondaryContentText">
+								Project Settings
+							</Text>
+						</Box>
+						{projectSettingTabs.map((project) => (
+							<NavLink
+								key={project.key}
+								to={`/${project.key}/settings/recording`}
+								className={clsx(styles.menuItem, {
+									[styles.menuItemActive]:
+										projectId === project.key,
+								})}
+							>
+								<Stack direction="row" align="center" gap="4">
+									<Text>{project.title}</Text>
+								</Stack>
+							</NavLink>
+						))}
+					</Stack>
```

Comment:
in the [figma](https://www.figma.com/file/rdBPNzn7Klk6RG1LHUCE5B/Screen-Designs?type=design&node-id=7552%3A287169&t=CUu6aswgUI0QCnpi-1), this also has a color button on the side. could we add that?

Reply:
What color should it be? Just some random color?

Reply:
could we use the project name hashed to an int to look up from our color pallet? similar to the avatar code https://github.com/highlight/highlight/blob/09c50163505c36bcdeaaed265722c4cd0cc684a1/frontend/src/components/Avatar/Avatar.tsx#L78